doctor/doctors.js

```diff
@@ -2,25 +2,82 @@ const { Doctor } = require("./doctor");
 
 exports.getDoctors = async (req, res) => {
   try {
+    if (req.query) {
+      return getDoctorByQuery(req, res);
+    }
+
+    let response = await Doctor.getDoctors();
+    
+    res
+      .code(response.statusCode)
+      .send(response);
   } catch (err) {}
 };
 
-exports.getDoctorByEmail = async (req, res) => {
+const getDoctorByQuery = async (req, res) => {
```

Comment:
export this

Reply:
Actually no need to:
- first, you cant have two endpoints with same structure
- second, you can hit getDoctors either with query-strings or, without query-strings